src/containers/viewer/ViewerContainer.ts

```diff
@@ -110,6 +93,32 @@ export const ViewerContainer = createContainer<ViewerValue>(() => {
     return () => worker.unsubscribe(WorkerEventType.RESULT_CALC, listener);
   }, [worker, current]);
 
+  useEffect(() => {
+    const sync = (location: Location) => {
+      const query = qs.parse(location.search, { decode: false });
```

Comment:
We can  'query-string' dependency as the following ?

```ts
      const query = new URLSearchParams(location.search);
      const id = query.get('id');
```

Reply:
See also: https://url.spec.whatwg.org/#interface-urlsearchparams

Reply:
I've forgotten about `URLSearchParams` :innocent:
I remove `query-string` from dependencies and replace the implementation with it!

Reply:
Fixed by https://github.com/reg-viz/reg-cli-report-ui/pull/40/commits/fe082fb06171b78ecd9c08d476dda064b72a29cb